aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>Prerequisites is one word.</p><p> </p><p>What do you mean by "Classic Jupyter mode"? Is there a link to docs that show you how to switch to this?</p><p> </p><p>Once you add periods to your list items, do so consistently (Open the listing...).</p><p> </p><p>Arn --&gt; ARN.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>